doc/plugin-develop.md

```diff
@@ -295,7 +295,188 @@ end
 
 ## implement the logic
 
-Write the logic of the plugin in the corresponding phase.
+Write the logic of the plugin in the corresponding phase. There are two parameters `conf` and `ctx` in the `phase` method, take the `limit-conn` plugin configuration as an example.
+
+```shell
+curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
+{
+    "methods": ["GET"],
+    "uri": "/index.html",
+    "id": 1,
+    "plugins": {
+        "limit-conn": {
+            "conn": 1,
+            "burst": 0,
+            "default_conn_delay": 0.1,
+            "rejected_code": 503,
+            "key": "remote_addr"
+        }
+    },
+    "upstream": {
+        "type": "roundrobin",
+        "nodes": {
+            "39.97.63.215:80": 1
+        }
+    }
+}'
+```
+
+### conf parameter
+
+The `conf` parameter caches the relevant configuration information of the plugin. The `conf` data obtained by `core.log.warn(core.json.encode(conf))` is as follows.
+
+```lua
+function _M.access(conf, ctx)
+    core.log.warn("conf: ", core.json.encode(conf))
+    ......
+end
+```
+
+conf:
+
+```json
+{
+    "rejected_code":503,
+    "burst":0,
+    "default_conn_delay":0.1,
+    "conn":1,
+    "key":"remote_addr"
+}
+```
+
+### ctx parameter
+
+The `ctx` parameter caches the data information related to the request. The `ctx` data obtained by `core.log.warn(core.json.encode(ctx, true))` is as follows.
+
+```lua
+function _M.access(conf, ctx)
+    core.log.warn("ctx: ", core.json.encode(ctx, true))
+    ......
+end
+```
+
+ctx:
+
+```json
+{
+    "matched_upstream":{
```

Comment:
This value will change across versions. And it also depends on where you log it. We just need to tell users how to check it.

Reply:
updated.